### DISCUSSION TITLE
Error 500 when trying to update/add theme

### DISCUSSION THREAD

**Discussion by brayozz** [JulianPrieber/littlelink-custom#505] [Questions]
Hi,

I'm getting error 500 whenever I try to upload a new theme or update the existing ones. 

Using LinkStack 4.1.2~ynh2 (Yunohost installer) on a Oracle machine VM.Standard.A1.Flex with 4 CPU's and 24 gb of memory. 

Does anybody have any solution for this case? Thank you in advance.

**Comment by JulianPrieber** [comment]
Please enable debugging in your config. You might have to increase the PHP upload size.
